examples/webpack/example-builder.js

```diff
@@ -12,6 +12,8 @@ const readFile = promisify(fs.readFile);
 const isCssRegEx = /\.css$/;
 const isJsRegEx = /\.js(\?.*)?$/;
 const importRegEx = /^import .* from '(.*)';$/;
+const isTemplateJs = /\/(jquery|(bootstrap(\.bundle)?))(\.min)?\.js$/;
```

Comment:
To find the jquery scripts as we include them, the regex should look like this:
```suggestion
const isTemplateJs = /\/(jquery(-[0-9.]+)?|(bootstrap(\.bundle)?))(\.min)?\.js$/;
```